packages/dds/tree/src/simple-tree/api/tree.ts

```diff
@@ -101,6 +102,26 @@ export interface ViewableTree {
  */
 export interface ITree extends ViewableTree, IFluidLoadable {}
 
+/**
+ * {@link ITree} extended with some alpha APIs.
+ * @privateRemarks
+ * Promote this to alpha.
+ * @internal
+ */
+export interface ITreeAlpha extends ITree {
+	/**
+	 * Exports root in the same format as {@link TreeAlpha.(exportVerbose:1)} using stored keys.
+	 */
+	exportVerbose(): VerboseTree | undefined;
```

Comment:
Under what circumstances will this yield `undefined`?

Reply:
done.